rdmo/core/constants.py

```diff
@@ -81,15 +81,15 @@
 }
 
 RDMO_MODELS = {
-  'catalog': 'questions.catalog',
-  'section': 'questions.section',
-  'page': 'questions.page',
-  'questionset': 'questions.questionset',
-  'question': 'questions.question',
   'attribute': 'domain.attribute',
-  'optionset': 'options.optionset',
-  'option': 'options.option',
   'condition': 'conditions.condition',
```

Comment:
Options before conditions.

Reply:
should be resolved by the revert to the original ordering, via the recursive `append_element`, function